### PR TITLE
Origin/sushmita/left toolbar highlight hospital pins

### PR DESCRIPTION
**Pull request by Sushmita-Ghosh**
## Description

On clicking the hospital card on the left toolbar , the corresponding hospital marker pin should be highlighted on the map.

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

- Related Issue # https://projects.digitalaidseattle.org/project-tasks-details?recordId=recnMJOVFdSP4F5Mg

## QA Instructions, Screenshots, Recordings

When any of the hospital card is clicked on the left toolbar, the corresponding pin color will turn yellow & the size will increase. If clicked again, the color & size go back to normal. 

When clicked/ highlighted:
image

When clicked again/ highlighting removes:
image

### CODE REVIEW

**Comment by Sushmita-Ghosh** [comment]
When the highlight gets added - I have changed the color to yellow & increased the size to denote a highlight. If any of these changes need to changed. Please let me know. Thanks!

**Comment by jnakaso** [comment]
I would add two feature:

1. Center the map on the selected hospital
2. Make the selection exclusive, in other words only the recent selection is highlighted.

**Review by jnakaso — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -51,6 +57,24 @@ function App() {
   }, []);
 
 
+  useEffect(() => {
+    if (hospitals.length > 0) {
```
[the remark this answers is not in the document]
Comment (Sushmita-Ghosh) [reply src/App.tsx]: Hi Jeff, I have gone through the changes &  have merged your branch that fixes the features/changes that you had requested. Let me know if there are any pending changes you would like to have for this PR.

**Review by jnakaso — APPROVED** [review]
Looks really good.  Thanks!

File: src/components/GFLMap.tsx [hunk 2]
```diff
@@ -42,19 +46,51 @@ export const GFLMap: React.FC<MapProps> = ({
...
-          longitude={hospital.longitude}
-          latitude={hospital.latitude}
-          onClick={() =>
-            setPopupInfo({
-              hospitalInfo: hospital,
-            })
-          }
-        />
-      ))}
+      {hospitals.map((hospital) => {
```
[the remark this answers is not in the document]
Comment (jnakaso) [reply src/components/GFLMap.tsx:56]: We can address this later.

**Pull request merged**